ThunderCatsTests/UIColorTests.m

```diff
@@ -81,6 +81,49 @@ - (void)testColorWithHexString
     XCTAssertFalse(r1-r2 == 0 && g1-g2 == 0 && b1-b2 == 0 && a1-a2 == 0);
 }
 
+- (void)testColorWithShortHexString
+{
+    UIColor *color1 = [UIColor tc_colorWithHexString:@"000"];
+    UIColor *color2 = [UIColor blackColor];
+    
+    CGFloat r1, r2, g1, g2, b1, b2, a1, a2;
+    [color1 getRed:&r1 green:&g1 blue:&b1 alpha:&a1];
+    [color2 getRed:&r2 green:&g2 blue:&b2 alpha:&a2];
+    
+    XCTAssertTrue(r1-r2 == 0 && g1-g2 == 0 && b1-b2 == 0 && a1-a2 == 0);
```

Comment:
I copied what was already in the file, can you look at the actual file and tell me if the whole thing needs to be redone?